Snehil/number_guessing_game.py

```diff
@@ -0,0 +1,22 @@
+print("hello , let's start the game ")
+from random import randint
+choice = randint(1, 100)
+print(choice)
+count = 5
+
+
+while(True and count):
```

Comment:
Imported libraries will come at the top of the code for better organisation. Also, the code doesn't check for invalid inputs. A user can put a alphanumeric value and the code will break.